index.js

```diff
@@ -146,7 +146,7 @@ app.use(
 );
 app.use('/v1/models', modelsRoutes);
 app.use('/v1/chat', chatRoutes);
-app.use('/v1/embeddings', embeddingsRoutes);
+app.use(/^\/v1(?:\/.+)?\/embeddings$/, embeddingsRoutes);
```

Comment:
what's the purpose of this change?

Reply:
It is in my PR comment:

index.js changed app.use('/v1/embeddings', embeddingsRoutes); to app.use(/^/v1(?:/.+)?/embeddings$/, embeddingsRoutes); to process embeddings like: https://api.openai.com/v1/engines/text-embedding-ada-002/embeddings

Auto-gpt also uses those API calls.